docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -93,7 +93,11 @@ Model Optimizer produces an Intermediate Representation (IR) of the network, whi
         * [Converting Your ONNX* Model](prepare_model/convert_model/Convert_Model_From_ONNX.md)
             * [Converting Faster-RCNN ONNX* Model](prepare_model/convert_model/onnx_specific/Convert_Faster_RCNN.md)
             * [Converting Mask-RCNN ONNX* Model](prepare_model/convert_model/onnx_specific/Convert_Mask_RCNN.md)
-            * [Converting DLRM ONNX* Model](prepare_model/convert_model/onnx_specific/Convert_DLRM.md)
+            * [Converting GPT2 ONNX* Model](prepare_model/convert_model/onnx_specific/Convert_GPT2.md)
```

Comment:
What was a framework used for training these onnx models?

Reply:
It is not relevant here. The models came as .onnx and we do not cover conversion to onnx in this section. So they should remain as onnx models.